### ISSUE TITLE
Missing `module.properties` entry `module.description` results in invalid amp

### ISSUE THREAD

**Issue by todorinskiz** [xenit-eu/alfresco-gradle-sdk#69]
Creating a vanila AMP, as is out-of-the-box from `eu.xenit.amp`, without specifying project description results in invalid AMP:

`07250000 The following module properties need to be defined: [module.description]`

Shouldn't we fallback to an empty string?

**Comment by thijslemmens** [comment]
07250000 The following module properties need to be defined: [module.description]

 Is that the error you get when building the amp? Because in that case, I think that's pretty reasonable.

**Comment by vierbergenlars** [comment]
No, you get that error when *applying* the AMP.

See also #56 for failing the build when your AMP is missing properties, which I prefer above letting you get away with not setting a description.

**Comment by thijslemmens** [comment]
I agree, the build should fail with a clear message. An empty description is not clean.
